.github/workflows/build-docker.yml

```diff
@@ -0,0 +1,68 @@
+name: Build Docker image
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+env:
+  ARM_GNU_TOOLCHAIN_VERSION: 14.2.rel1
+  ARM_GNU_TOOLCHAIN_PLATFORM: x86_64-arm-none-eabi
+  EM_VERSION: 4.0.1
+  EM_NODE_VERSION: 20.18.0_64bit
```

Comment:
should likely be defined just in one place (Dockerfile)

Reply:
I put all the require envvars in Dockerfile